Data-Science/Ray/Ray-CPU/Matrix_Multiplication/README.md

```diff
@@ -1,44 +1,49 @@
-## Ray Fibonacci Example
+## Ray Matrix_Multiplication Example
 
 ##### This example demonstrates the comparison between executing tasks locally and distributedly using Ray.
 
 ### Prerequisites:
-* Ensure you are using a Data Science notebook environment in Kubeflow. 
+* Ensure you are using a Tensorflow notebook environment with aleast 3 CPUs and 4 Gi of memory in Kubeflow. 
```

Comment:
Why do we need to choose Tensorflow notebook? Anything missing with Data Science notebook?

Reply:
We can use any notebook image

Reply:
Changed to Data Science